src/pages/HomePage/pages/LifeCenter/LifeCenter.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useState } from "react";
+// import PageOutline from "../../Components/PageOutline";
+import TabSelection from "@/pages/HomePage/Components/reusable/TabSelection";
+import { Button } from "@/components";
+import Modal from "@/components/Modal";
+import { HeaderControls } from "@/components/HeaderControls";
+import LifeCenterForm from "./components/LifeCenterForm";
+import LifeCenterCard from "./components/LifeCenterCard";
+import useWindowSize from "@/CustomHooks/useWindowSize";
+import { useFetch } from "@/CustomHooks/useFetch";
+import { api } from "@/utils/api/apiCalls";
+import GridComponent from "../../Components/reusable/GridComponent";
+
+export type LifeCenterType = {
+  name: string;
+  description: string;
+  location: string;
+  meeting_dates: string[];
+  num_of_members: number;
+  num_of_souls_won: number;
+  id?: string;
+};
+
+export default function LifeCenter() {
+  const [selectedTab, setSelectedTab] = useState("Life Center");
+  const [showModal, setShowModal] = useState(false);
+  const windowSize = useWindowSize();
+  const { data } = useFetch(api.fetch.fetchAllLifeCenters);
+
+  const handleTabSelect = (tab: string) => {
+    setSelectedTab(tab);
+  };
+
+  const [lifeCenters, setLifcenters] = useState<LifeCenterType[]>([]);
```

Comment:
typo in your code _setLifeCenters_ will be really annoying to spot if autocomplete is not working